index.html

```diff
@@ -232,6 +232,59 @@ <h5 class="text-white">Skills</h5>
           </div>
         </div>
       </section>
+      <section class="contact-form-section container">
+      <div class="contact-form-title-section">
+        <h4 class="contact-form-title font-inter">
+          I'm always interested in hearing about new projects, so if you'd
+          like to chat please get in touch.
+        </h4>
+      </div>
+      <form action="https://formspree.io/f/mzborevw" method="post" class="contact-form">
+        <label class="hide-label" for="name">Full Name</label>
+        <input type="text" name="name" class="input-field font-inter" id="name" placeholder="Full name" maxlength="30"
+          required />
+        <label class="hide-label" for="email">Email Address</label>
+        <input type="email" name="email" class="input-field font-inter mt-24" id="email" placeholder="Email address"
+          required />
+        <label class="hide-label" for="message">Your message</label>
+        <textarea name="message" class="input-field input-textarea font-inter mt-24" id="message" maxlength="500"
+          required>Enter text here</textarea>
+        <button class="btn mt-24">Get in touch</button>
+      </form>
+    </section>
+    <hr/>
+    <footer>
+      <ul class="footer-social-media-links">
+        <li>
+          <a href="#" class="footer-social-media-icon">
+            <img src="assets/svg/github.svg" alt="Github icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#" class="footer-social-media-icon">
+            <img src="assets/svg/linkedin.svg" alt="Linkedin icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#" class="footer-social-media-icon">
+            <img src="assets/svg/angel-list.svg" alt="Angel List icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#" class="footer-social-media-icon">
+            <img src="assets/svg/twitter.svg" alt="Twitter icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#" class="footer-social-media-icon">
+            <img src="assets/svg/medium.svg" alt="Medium Icon" />
+          </a>
+        </li>
+      </ul>
+      <div class="footer-line-section">
+        <img src="assets/png/footer-line-shape.png" alt="Footer Line Shape">
+      </div>
+    </footer>
```

Comment:
[OPTIONAL] It would be better to change the color of social icons to a dark color to improve your UI design:
- Your design 
<img width="205" alt="image" src="https://user-images.githubusercontent.com/77190203/159579074-8996ffa7-20b0-4155-8dce-8e0625943347.png">

- Figma design:
<img width="311" alt="image" src="https://user-images.githubusercontent.com/77190203/159579370-6a95b852-0892-47a4-b86d-57623cd214ca.png">